src/styles/main.scss

```diff
@@ -1,3 +1,66 @@
+@import "variables";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: center;
+  font-family: $font-family;
+}
+
+.calendar {
+  display: flex;
+  width: 710px;
+  flex-wrap: wrap;
+  height: min-content;
+  gap: 1px;
+
+  &__day {
+    display: flex;
+    width: $day_size;
+    height: $day_size;
+    justify-content: center;
+    align-items: center;
+    background-color: $day_background;
+    border: 1px solid $border-color;
+    font-size: 30px;
+    transition: all 0.5s;
+
+    &:hover {
+      background-color: $background_pink;
+      cursor: pointer;
+      transform: translateY(-20px);
+      transition: all 0.5s;
+    }
+  }
+}
+
+@each $day, $margin in $start_day {
+  .calendar--start-day-#{$day} {
+    > .calendar__day:first-child {
+      margin-left: $margin;
+      counter-reset: day-counter;
+    }
+
+    > .calendar__day::before {
+      content: counter(day-counter);
+      counter-increment: day-counter;
+    }
+  }
+}
+
+@for $i from 28 through 31 {
+  .calendar--month-length-#{$i} {
+    > div {
+      &:nth-child(n + #{$i + 1}) {
+        display: none;
+      }
+    }
+  }
```

Comment:
move all these styles to calendar selector and use sass nesting
https://sass-lang.com/documentation/style-rules/parent-selector/#adding-suffixes

Reply:
<img width="1234" alt="Screen Shot 2023-10-25 at 2 51 50 PM" src="https://github.com/mate-academy/layout_calendar/assets/139006847/e71eaf5b-5879-4b5a-a834-25701f429951">
<img width="1220" alt="Screen Shot 2023-10-25 at 2 52 06 PM" src="https://github.com/mate-academy/layout_calendar/assets/139006847/d04e7494-8965-4021-9570-7ddd25ec5990">
<img width="1132" alt="Screen Shot 2023-10-25 at 2 52 17 PM" src="https://github.com/mate-academy/layout_calendar/assets/139006847/e26462ab-2097-4904-8dd7-0a13c3dc6cfe">

Reply:
I'm sorry, but my code isn't working despite trying suggested changes. I couldn't find a solution on my own, with mentors, or with chat GPT. If you could kindly help to solve this issue, that would be greatly appreciated.